src/modules/core/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,51 @@
+import { type FC } from 'react';
+import clsx from 'clsx';
+import Image from 'next/image';
+// components
+import { Icon } from '@/modules/core/components';
+
+import type { AvatarProps, AvatarSize } from './avatar.interface';
+import styles from './avatar.module.scss';
+
+const SIZES: Record<AvatarSize, number> = {
+  small: 40,
+  medium: 56,
+  large: 156,
+};
+
+export const Avatar: FC<AvatarProps> = ({
+  url,
+  size = 'small',
+  fallback = <Icon name='user' className={styles.userIcon} />,
+  shadow = false,
+  shape = 'circle',
+}) => {
+  const avatarSize = SIZES[size];
```

Comment:
I think we can even go further and remove this const, and also remove return, that would look nicer.